app/js/service/rate.js

```diff
@@ -12,7 +12,7 @@
   const USE_CORS_PROXY = RSBP_CONFIG.rate.useCorsProxy;
   const EXPIRATION = RSBP_CONFIG.rate.expiration;
   const IS_BTC = CURRENCY === "BTC";
-  const URL = "https://api.bitcoinaverage.com/ticker/" + CURRENCY + "/last";
+  const URL = "https://apiv2.bitcoinaverage.com/indices/global/ticker/" + CURRENCY + "/last";
```

Comment:
I think line 15 still needs some changes. For example, if CURRENCY is IDR, URL would become "https://apiv2.bitcoinaverage.com/indices/global/ticker/IDR/last", which gives a 404 error